src/templates/Admin/config.js

```diff
@@ -48,4 +49,11 @@ export const config = {
     CaseStudyConfig,
     BlogPostConfig,
   ],
+  media_library: {
+    name: 'cloudinary',
+    config: {
+      cloud_name: `${CLOUDINARY_CLOUD_NAME}`,
```

Comment:
You can avoid to use the tagged template since there's no concatenation
```suggestion
      cloud_name: CLOUDINARY_CLOUD_NAME,
```